viewer/vueapp/src/store.js

```diff
@@ -217,10 +218,24 @@ const store = new Vuex.Store({
       state.user = value.user;
       state.views = value.views;
       state.fieldsArr = value.fieldsArr;
-      state.fieldsMap = value.fieldsMap;
       state.remoteclusters = value.remoteclusters;
       state.fieldhistory = value.fieldhistory.fields || [];
       state.esCluster.availableCluster = value.clusters;
+
+      // fieldsMap has keys for these fields: dbField, dbField2, fieldECS, and exp (id/key)
+      // fieldsAliasMap has keys for field aliases
+      for (const key in value.fieldsMap) {
+        const field = value.fieldsMap[key];
+        state.fieldsMap[field.exp] = field;
+        state.fieldsMap[field.dbField] = field;
+        state.fieldsMap[field.dbField2] = field;
```

Comment:
dbField2 can be undefined if you want to check for it

Reply:
Got it, should I also fix this for `viewer/config.js`?
```
internals.fieldsMap[field._id] = source;
internals.dbFieldsMap[source.dbField] = source;
internals.dbFieldsMap[source.dbField2] = source;
```
(in the `loadFields` function)

Reply:
sure!

Reply:
👍